typescript/sdk/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "The official SDK for the Hyperlane Network",
   "version": "4.1.0",
   "dependencies": {
+    "@arbitrum/sdk": "^3.5.1",
```

Comment:
I guess this is another instance where it might be good to have the relayer in a separate package so as not to pollute the SDK dependency tree with these

Reply:
Good point, here is the issue for this: https://github.com/hyperlane-xyz/issues/issues/1340